src/api/app/models/bs_request_action/errors.rb

```diff
@@ -7,7 +7,10 @@ class RemoteSource < APIError
     setup 'remote_source', 400, 'No support for auto expanding from remote instance. You need to submit a full specified request in that case.'
   end
   class RemoteTarget < APIError; end
-  class InvalidReleaseTarget < APIError; end
+  class InvalidReleaseTarget < APIError
+    setup 'invalid_release_target', 400, 'Can not release to a maintenance incident project'
+  end
+  class MultipleReleaseTargets < APIError; setup 'Multiple release target projects are not supported'; end
```

Comment:
A `400` error code should be added here too. Isn't it?

Reply:
just if it's used in the controllers, right?

Reply:
So then in line 11 for `InvalidReleaseTarget`, the `400` can also be removed? Or is it another use case?